pkg/apis/extensions/v1alpha1/rpaasplan_types.go

```diff
@@ -81,6 +85,21 @@ type NginxConfig struct {
 	WorkerConnections int `json:"workerConnections,omitempty"`
 }
 
+type CacheHeaterSyncSpec struct {
+	// The schedule in Cron format, see https://en.wikipedia.org/wiki/Cron.
+	Schedule string `json:"schedule,omitempty"`
+
+	// Container image used to sync the containers
+	// default is bitnami/kubectl:latest
+	Image string `json:"image,omitempty"`
```

Comment:
Since we let the image be customized I think we also need a `Cmds []string` here. Different could behave differently.

Reply:
yeah!, @cezarsa I've updated the code, thanks!